### PR TITLE
DOC: consistent imports (GH9886) part V

### PR DESCRIPTION
**Pull request by springcoil**
I made a few alterations to the consistent imports in the merging.rst file. 

Does anyone have any objections to what I did?

I notice a few problems with the rebase though first - that's 12 commits rather than one.

### CODE REVIEW

**Comment by springcoil** [comment]
Ok updated the commit by rebasing.

**Comment by springcoil** [comment]
Ok this should be all the changes done in the merging.rst file.

File: doc/source/whatsnew/v0.17.0.txt [hunk 1]
```diff
@@ -724,15 +724,40 @@ Performance Improvements
 
 Bug Fixes
 ~~~~~~~~~
-
 - Bug in incorrection computation of ``.mean()`` on ``timedelta64[ns]`` because of overflow (:issue:`9442`)
```
Comment (jreback) [inline doc/source/whatsnew/v0.17.0.txt]: remove all of these files that have conflicts

File: pandas/io/tests/generate_legacy_storage_files.py [hunk 2]
```diff
@@ -83,9 +83,20 @@ def create_data():
                             index=MultiIndex.from_tuples(tuple(zip(*[[1, 1, 2, 2, 2], [3, 4, 3, 4, 5]])),
                                                          names=['one', 'two'])),
                   dup=Series(np.arange(5).astype(np.float64), index=['A', 'B', 'C', 'D', 'A']),
+<<<<<<< HEAD
+<<<<<<< HEAD
                   cat=Series(Categorical(['foo', 'bar', 'baz'])))
     if LooseVersion(pandas.__version__) >= '0.17.0':
```
Comment (jreback) [inline pandas/io/tests/generate_legacy_storage_files.py]: here too

File: pandas/tests/test_generic.py [hunk 3]
```diff
@@ -799,7 +799,17 @@ def test_nan_interpolate(self):
         tm._skip_if_no_scipy()
         result = s.interpolate(method='polynomial', order=1)
         assert_series_equal(result, expected)
+ts
+    
+    # GH #10633
+    def test_interpolate_spline(self):
+        np.random.seed(1)
```
Comment (jreback) [inline pandas/tests/test_generic.py]: this too

File: pandas/tseries/tests/test_period.py [hunk 4]
```diff
@@ -2537,7 +2537,17 @@ def test_searchsorted(self):
 
     def test_round_trip(self):
 
+<<<<<<< HEAD
+<<<<<<< HEAD
         p = Period('2000Q1')
+=======
+        import pickle
+        p = Period('2000Q1')
+
```
Comment (jreback) [inline pandas/tseries/tests/test_period.py]: here as well

File: pandas/tseries/tests/test_timeseries.py [hunk 5]
```diff
@@ -923,8 +923,8 @@ def test_to_datetime_with_apply(self):
         assert_series_equal(result, expected)
 
         td = pd.Series(['May 04', 'Jun 02', ''], index=[1,2,3])
-        self.assertRaises(ValueError, lambda : pd.to_datetime(td,format='%b %y', errors='raise'))
-        self.assertRaises(ValueError, lambda : td.apply(pd.to_datetime, format='%b %y', errors='raise'))
+        self.assertRaises(ValueError, 'Unknown string format. You can coerce errors to NaT by passing coerce', lambda : pd.to_datetime(td,format='%b %y', errors='raise'))
```
Comment (jreback) [inline pandas/tseries/tests/test_timeseries.py]: here as well

**Comment by jreback** [comment]
@springcoil if behooves you to keep your master up to date and clean, e.g. don't make changes on it.

so

```
git checkout master
git reset upstream/master --hard
```

then when checking out

```
git checkout master
git pull
git checkout -b mybranch --track master
```

will keep things clean

**Comment by springcoil** [comment]
Reopening this branch.

**Comment by springcoil** [comment]
This is deprecated by [#10941](https://github.com/pydata/pandas/pull/10941)

**Pull request closed without merging**